pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator_helper.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
nit: you need not have the alias `apierrors`

Reply:
Why not? There's another `errors` package already imported in the file.

Reply:
I didn't see the full diff, my bad